README.md

```diff
@@ -89,7 +89,7 @@ The termination handler DaemonSet installs into your cluster a [ServiceAccount](
 
 You can use kubectl to directly add all of the above resources with the default configuration into your cluster.
 ```
-kubectl apply -f https://github.com/aws/aws-node-termination-handler/releases/download/v1.10.0/all-resources.yaml
+kubectl apply -f https://github.com/aws/aws-node-termination-handler/releases/download/v1.11.0/all-resources.yaml
```

Comment:
Can we leave off the readme update of the release automation?  Since NTH takes a while to release, at least 30-45 minutes, the readme will result in errors during that time. 

I think it's better to add a post-release step executed in Travis that uses ec2-bot to submit a PR back to the repo to update the README once all the assets are released (should be similar to the eks-charts sync script).

Reply:
Not opposed, but are we good with source code for a given release not having accurate version numbers in its ReadMe? Or were you thinking fewer users will be affected in this scenario vs. pulling a version that hasn't released yet

Reply:
Yeah, I think that including the readme w/ previous version in the archive files attached to the release is fine. We don't tell anyone to use those archives and I don't see a reason, even if they did, to use that readme as a source of truth.

Reply:
will submit a separate PR to update as a post-release step